### PR TITLE
Translate JP chapter 8

### PR DESCRIPTION
**Pull request by younesbelkada**
WIP PR - translate chapter 8

- [x] ```1.mdx```
- [x] ```2.mdx```
- [ ] ```3.mdx```
- [ ] ```4.mdx```
- [ ] ```5.mdx```
- [x] ```6.mdx```
- [ ] ```7.mdx```

Related issue: #114 
cc @lewtun @hiromu166 @HiromuHota @kojimano

### CODE REVIEW

File: chapters/ja/chapter8/1.mdx [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+🤗 Transformers　を使いながらNLPタスクに取り組む方法がわかった後、自分自身のプロジェクトを簡単に始めることができます!　この章では、エラーにぶつかったときにどうすればよいかを深く探ります。自分のコードやトレーニングを正確にデバッグする方法、そして自分でエラーを解決できない場合にコミュニティに助けを求める方法を一緒に学びましょう。また、HuggingFace ライブラリのひとつにバグを見つけたと思ったら、その問題をできるだけ早く解決するため報告する方法を紹介します。
+
+この章では、一緒に次のことを学びます：
+
+-　エラーを見つけた時に最初にすること
+-　[HuggingFace フォーラム](https://discuss.huggingface.co/)の中で助けの求め方
+-　トレーニングパイプラインのデバグ方法
+-　良いGitHubイシューの書き方
+
+もちろん、このどれもが🤗 TransformersやHuggingFacのエコシステムと特別な関係はありません。この章から得られる教訓は、ほとんどのオープンソースプロジェクトに適用可能です
```
Comment (HiromuHota) [inline chapters/ja/chapter8/1.mdx:12]: typo: "HuggingFac" -> "HuggingFace"

File: chapters/ja/chapter8/1.mdx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# イントロダクション
+
+🤗 Transformers　を使いながらNLPタスクに取り組む方法がわかった後、自分自身のプロジェクトを簡単に始めることができます!　この章では、エラーにぶつかったときにどうすればよいかを深く探ります。自分のコードやトレーニングを正確にデバッグする方法、そして自分でエラーを解決できない場合にコミュニティに助けを求める方法を一緒に学びましょう。また、HuggingFace ライブラリのひとつにバグを見つけたと思ったら、その問題をできるだけ早く解決するため報告する方法を紹介します。
```
Comment (HiromuHota) [inline chapters/ja/chapter8/1.mdx:3]: nit: "HuggingFace" -> "ハギングフェイス" as we discussed at https://discord.com/channels/879548962464493619/968862972527128629/985615240202944542?

File: chapters/ja/chapter8/2.mdx [hunk 3]
```diff
@@ -0,0 +1,361 @@
...
+    new_repo_id = get_full_repo_name(model_name)
+    new_repo_dir = model_name
+    repo = Repository(local_dir=new_repo_dir, clone_from=new_repo_id)
+    # Copy files
+    copy_tree(template_repo_dir, new_repo_dir)
+    # Push to Hub
+    repo.push_to_hub()
+```
+
+`copy_repository_template()`機能はテンプレートレポジトリーをアカンウトにコピーします。
```
Comment (HiromuHota) [inline chapters/ja/chapter8/2.mdx:54]: nit: アカンウト -> アカウント

File: chapters/ja/chapter8/2.mdx [hunk 4]
```diff
@@ -0,0 +1,361 @@
...
+
+## 🤗 Transformers　パイプラインのデバグ
+
+Transformerモデルとパイプラインのデバグという素晴らしい世界への旅を始めるにあたり、次のようなシナリオを考えてみましょう。あなたは同僚と一緒に、あるeコマースサイトに関するお客さんの答えを見つけられるように、'Question Answering'のプロジェクトに取り組んでいます。あなたの同僚はこんなメッセージを送りました。
```
Comment (HiromuHota) [inline chapters/ja/chapter8/2.mdx:58]: nit: デバグ -> デバッグ

File: chapters/ja/chapter8/2.mdx [hunk 5]
```diff
@@ -0,0 +1,361 @@
...
+💡 理解しがたいエラーメッセージが表示された場合は、そのメッセージをコピーしてGoogle または [Stack Overflow](https://stackoverflow.com/) の検索バーに貼り付けてください！　そのエラーに遭遇したのはあなたが初めてではない可能性が高いですし、コミュニティの他の人が投稿した解決策を見つける良い方法です。例えば、`OSError: Can't load config for` を Stack Overflow で検索すると、いくつかの [ヒント](https://stackoverflow.com/search?q=OSError%3A+Can%27t+load+config+for+) が見付けられます。これは、問題解決の出発点として使うことができます。
+</Tip>
+
+最初の提案は、モデルIDが実際に正しいかどうかを確認するよう求めているので、まず、識別子をコピーしてHubの検索バーに貼り付けましょう。
+
+<div class="flex justify-center">
+<img src="https://huggingface.co/datasets/huggingface-course/documentation-images/resolve/main/en/chapter8/wrong-model-id.png" alt="The wrong model name." width="100%"/>
+</div>
+
+うーん、確かに同僚のモデルはHubにないようだ...しかし、モデルの名前にタイプミスがある! DistilBERTの名前には「l」が1つしかないので、それを直して、代わりに「lewtun/distilbert-base-uncased-finetuned-squad-d5716d28」を探そう！
```
Comment (HiromuHota) [inline chapters/ja/chapter8/2.mdx:117]: Hub -> ハブ?

File: chapters/ja/chapter8/6.mdx [hunk 6]
```diff
@@ -0,0 +1,7 @@
+# 第2部終了!
+
+お疲れ様でした。第２部を無事に完了しました！今は第３部について働いてるので情報を見逃せないように我々の[ニュースレター](https://huggingface.curated.co/)に応募して下さい！
```
Comment (HiromuHota) [inline chapters/ja/chapter8/6.mdx:3]: 第２部 -> パート２ as discussed https://github.com/huggingface/course/pull/241#discussion_r899999768

**Review by HiromuHota — COMMENTED** [review]
Awesome work!
Left some suggestions.

**Comment by younesbelkada** [comment]
Thank you for your review!! I have fixed the issues now :)

**Review by lewtun — COMMENTED** [review]
Hey @younesbelkada thanks a lot for working on the Japanese translation of chapter 8 - the content so far looks great!

Would it make sense to merge the sections that you've currently translated and work on the rest in a separate PR? If yes, you'll just need to delete the English sections and update the `_toctree.yml` file accordingly :)

**Comment by younesbelkada** [comment]
Hi @lewtun !
Thanks for the review, that would work for me! 💪 
Let me know if the current change works for you

**Review by lewtun — APPROVED** [review]
Thanks for iterating on this @younesbelkada ! The current state looks great, so will merge and deploy today :)

**Pull request merged**